tests/basic/storage_pool_test.go

```diff
@@ -2053,4 +2052,574 @@ var _ = Describe("{VolUpdateAddDrive}", func() {
+			log.FailOnError(err, "Failed to get Pool IDs from the volume [%s]", poolUUIDs)
+
+			// Select the random pools from UUIDs for PxDriver Restart
+			randomIndex = rand.Intn(len(poolUUIDs))
+			rebootPoolID := poolUUIDs[randomIndex]
+
+			// Rebooting Node
+			log.InfoD("Get the Node for Restart %v", rebootPoolID)
+			restartDriver, err := GetNodeWithGivenPoolID(rebootPoolID)
+			log.FailOnError(err, "Geting Node Driver for restart failed")
+
+			isjournal, err := isJournalEnabled()
+			log.FailOnError(err, "Failed to check if Journal enabled")
+
+			poolToBeResized, err := GetStoragePoolByUUID(rebootPoolID)
+			log.InfoD("Pool to be resized %v", poolToBeResized)
+			log.FailOnError(err, "Failed to get pool using UUID [%s]", rebootPoolID)
+			expectedSize := poolToBeResized.TotalSize * 2 / units.GiB
+
+			log.InfoD("Restarting the Driver on Node [%s]", restartDriver.Name)
+			err = Inst().N.RebootNode(*restartDriver, node.RebootNodeOpts{
+				Force: true,
+				ConnectionOpts: node.ConnectionOpts{
+					Timeout:         1 * time.Minute,
+					TimeBeforeRetry: 5 * time.Second,
+				},
+			})
+			log.FailOnError(err, "Rebooting Node failed ?")
+
+			log.InfoD("Waiting till Volume is In Resync Mode ")
+			if WaitTillVolumeInResync(randomVolIDs) == false {
+				log.InfoD("Failed to get Volume in Resync state [%s] ", randomVolIDs)
+			}
+
+			log.InfoD("Current Size of the pool %s is %d", rebootPoolID, poolToBeResized.TotalSize/units.GiB)
+			err = Inst().V.ExpandPool(rebootPoolID, api.SdkStoragePool_RESIZE_TYPE_AUTO, expectedSize)
+			dash.VerifyFatal(err, nil, "Pool expansion init successful?")
+
+			resizeErr := waitForPoolToBeResized(expectedSize, rebootPoolID, isjournal)
+			dash.VerifyFatal(resizeErr, nil, fmt.Sprintf("Verify pool [%s] on node [%s] expansion using auto", rebootPoolID, restartDriver.Name))
+
+		}
+		opts := make(map[string]bool)
+		opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+		ValidateAndDestroy(contexts, opts)
+
+		Step("destroy all the applications created before test runs", func() {
+			opts := make(map[string]bool)
+			opts[scheduler.OptionsWaitForResourceLeakCleanup] = true
+			for _, ctx := range contexts {
+				TearDownContext(ctx, opts)
+			}
+		})
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		AfterEachTest(contexts, testrailID, runID)
+	})
+})
+
+var _ = Describe("{PoolIncreaseSize20TB}", func() {
+	var testrailID = 51292
+	// Testrail Description : Resize a pool of capacity of 100GB to 20TB
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/51292
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolIncreaseSize20TB", "Resize a pool of capacity of 100GB to 20TB", nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	//var vol_ids []string
+	stepLog := "should get the existing storage node and expand the pool by resize-disk"
+	It(stepLog, func() {
```

Comment:
Move `steLog` message inside `It("should get the existing storage node and expand the pool by resize-disk", func() {})`